tests/logs.rs

```diff
@@ -40,7 +40,7 @@ fn cleanup_files(stats_log_file: &str, debug_log_file: &str) {
 }
 
 #[test]
-fn test_stat_logs_json() {
+fn test_stats_logs_json() {
```

Comment:
Fixed naming as part of this PR as these tests regressed and I was running them independently to test the fix. By starting them with a common "stats_logs" instead of "stat_logs" it's a single command to re-run them all together.

Reply:
👍